app/controllers/candidate_interface/gcse/type_controller.rb

```diff
@@ -61,9 +63,25 @@ def qualification_params
         .merge!(
           subject: subject_param,
           level: ApplicationQualification.levels[:gcse],
+          grade: current_qualification&.grade,
+          award_year: current_qualification&.award_year,
+          institution_name: current_qualification&.institution_name,
+          institution_country: current_qualification&.institution_country,
```

Comment:
do you need to merge in not completed explanation and missing information here too? Otherwise they will be update to nil when you update right?